cmd/kubelet/app/server.go

```diff
@@ -373,6 +374,10 @@ func UnsecuredDependencies(s *options.KubeletServer) (*kubelet.Dependencies, err
 func Run(s *options.KubeletServer, kubeDeps *kubelet.Dependencies) error {
 	// To help debugging, immediately log version
 	glog.Infof("Version: %+v", version.Get())
+	// This can happen only on Windows
```

Comment:
Not sure this comment is necessary; I could see `initForOS` growing to do other OS-specific non-Windows things in the future.